src/textual/widget.py

```diff
@@ -2421,6 +2455,18 @@ def release_mouse(self) -> None:
         """
         self.app.capture_mouse(None)
 
+    def check_message_enabled(self, message: Message) -> bool:
```

Comment:
`MessagePump.check_message_enabled` is missing a docstring and so is this method, so we get no docstring here.
Not sure if you feel like adding a docstring to `MessagePump` _but_ I think it's worth adding a docstring here.

Reply:
I think I'll also leave this for a subsequent round of "add more documentation", rather than make this PR about adding previously-missing docs.

Reply:
Added to #1811 as a reminder.

Reply:
Right, but the point was that even if you document the super class's method, this method probably needs an extended docstring either way, no?

Reply:
I think I'd need to see the proper documentation for the super-method to decide of the additional test that happens in `Widget` is in or out of the scope, and either way I'd need to see the documentation for the super-method to know how this one should best be documented (hence a note to document it here too in #1811).